test/runnable/testdstress.d

```diff
@@ -669,22 +669,6 @@ void test30()
 
 /* ================================ */
 
-void test31()
-{
-        string str = x"F0 9D 83 93"; // utf-8 for U+1D0D3
```

Comment:
ditto

Reply:
Right,  I may have deleted way too much here without realizing that it wasn't the hex string literal that was under test, but something else.  I'll update all of these soon.